OmsiHook/OmsiStructs.cs

```diff
@@ -1832,4 +1878,22 @@ public struct RTL_CRITICAL_SECTION
         public uint SpinCount;
     }
 
+    public struct OmsiWString
```

Comment:
Maybe this `OmsiFloatPtr` should get a little docstring explaining what they're for.
Also the thing I said earlier about giving them a couple of constructors.